InfiniTAM/InfiniTAM_node.cpp

```diff
@@ -124,6 +132,13 @@ int main(int argc, char** argv) try {
   ros::init(argc, argv, "infinitam_node");
   ros::NodeHandle node_handle("~");
 
+  ITMMainEngine* main_engine = NULL;
```

Comment:
use `nullptr` instead of `NULL` (see e.g. http://stackoverflow.com/questions/13816385/what-are-the-advantages-of-using-nullptr)